scheduler-interface-spec.md

```diff
@@ -826,6 +826,37 @@ message ForgotAllocation {
 }
 ```
 
+#### Event Cache
+
+The Event Cache is a SchedulerPlugin that exposes events about scheduler objects aiming to help the end user to
+see these events from the shim side. An event is sent to the shim side through the callback in a form of an `EventMessage`.
+
+```protobuf
+message EventRecord {
```

Comment:
what is it `EventMessage` or `EventRecord` ?
these need to be in sync

Reply:
Right, forgot to modify it. Fixed.